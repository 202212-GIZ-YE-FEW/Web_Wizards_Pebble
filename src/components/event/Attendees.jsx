import SingleAttendees from "./SingleAttendees";
import Title from "./Title";
const Attendees = (props) => {
    const eventData = props.eventData;

    return (
        <div className='order-2 lg:w-1/3 w-full lg:ml-12 py-8'>
            <Title text='Attendees:' />
            <div className='flex flex-wrap lg:justify-start'>
                {eventData?.attendees?.map((attende, index) => (
                    <SingleAttendees key={index} name={attende} />
                ))}
            </div>
        </div>
    );
};

export default Attendees;
