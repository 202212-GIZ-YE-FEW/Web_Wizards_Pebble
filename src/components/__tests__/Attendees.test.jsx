import renderer from "react-test-renderer";

import Attendees from "@/components/event/Attendees";

it("renders correctly", () => {
    const eventData = {
        title: "Meeting Brainstorm Energy CIeccan lzmir",
        location: "Izmir, Turkey",
        date: "ahhallsei 26 Augest Kapisi",
        organizer: "Sufyan",
        attendees: ["Sufyan", "Adeeb", "Abdullah", "Azzam", "Hamad"],
    };
    const tree = renderer.create(<Attendees eventData={eventData} />).toJSON();
    expect(tree).toMatchSnapshot();
});
