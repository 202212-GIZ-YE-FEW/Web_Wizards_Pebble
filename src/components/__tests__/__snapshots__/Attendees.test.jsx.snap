// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="order-2 lg:w-1/3 w-full lg:ml-12 py-8"
>
  <h2
    className="text-2xl font-semibold mb-2 "
  >
    Attendees:
  </h2>
  <div
    className="flex flex-wrap lg:justify-start"
  >
    <div
      className="py-2 w-1/3 md:w-1/6 lg:w-1/4"
    >
      <div
        className="flex items-center justify-center w-16 h-16 rounded-full bg-black-100 text-white text-md font-bold mb-1"
      >
        <span>
          S
        </span>
      </div>
      <p
        className="text-sm font-semibold"
      >
        Sufyan
      </p>
    </div>
    <div
      className="py-2 w-1/3 md:w-1/6 lg:w-1/4"
    >
      <div
        className="flex items-center justify-center w-16 h-16 rounded-full bg-black-100 text-white text-md font-bold mb-1"
      >
        <span>
          A
        </span>
      </div>
      <p
        className="text-sm font-semibold"
      >
        Adeeb
      </p>
    </div>
    <div
      className="py-2 w-1/3 md:w-1/6 lg:w-1/4"
    >
      <div
        className="flex items-center justify-center w-16 h-16 rounded-full bg-black-100 text-white text-md font-bold mb-1"
      >
        <span>
          A
        </span>
      </div>
      <p
        className="text-sm font-semibold"
      >
        Abdullah
      </p>
    </div>
    <div
      className="py-2 w-1/3 md:w-1/6 lg:w-1/4"
    >
      <div
        className="flex items-center justify-center w-16 h-16 rounded-full bg-black-100 text-white text-md font-bold mb-1"
      >
        <span>
          A
        </span>
      </div>
      <p
        className="text-sm font-semibold"
      >
        Azzam
      </p>
    </div>
    <div
      className="py-2 w-1/3 md:w-1/6 lg:w-1/4"
    >
      <div
        className="flex items-center justify-center w-16 h-16 rounded-full bg-black-100 text-white text-md font-bold mb-1"
      >
        <span>
          H
        </span>
      </div>
      <p
        className="text-sm font-semibold"
      >
        Hamad
      </p>
    </div>
  </div>
</div>
`;
